septentrion/cli.py

```diff
@@ -111,6 +111,23 @@ def split_envvar_value(self, rv: str):
     "immediately if it doesn't exist (env: SEPTENTRION_TABLE)",
     default=configuration.DEFAULTS["table"],
 )
+@click.option(
+    "--version_column",
+    help="Version column name in the migration table. "
+    "(env: SEPTENTRION_VERSION_COLUMN)",
+    default=configuration.DEFAULTS["version_column"],
+)
+@click.option(
+    "--name_column",
+    help="Name column name in the migration table. " "(env: SEPTENTRION_NAME_COLUMN)",
+    default=configuration.DEFAULTS["name_column"],
+)
+@click.option(
+    "--applied_column",
+    help="Applied at column name in the migration table. "
+    "(env: SEPTENTRION_APPLIED_COLUMN)",
```

Comment:
```suggestion
    help="Name of the column describing the date at which the migration was applied "
    "in the migrations table. (env: SEPTENTRION_APPLIED_COLUMN)",
```
Also, can we name it `applied_at` rather than `applied` which sound like a boolean?